src/components/pager/pager.style.ts

```diff
@@ -15,10 +15,9 @@ interface StyledPagerProps {
 const StyledSelectContainer = styled.div`
   height: 26px;
   width: 55px;
-  margin-left: 4px;
-  margin-right: 4px;
-
-  ${StyledInputPresentation} {
+  margin-left: 8px;
+  margin-right: 8px;
+  s ${StyledInputPresentation} {
```

Comment:
**question(non-blocking):** I guess this is a mistake or is `s` for space? :trollface: 
```suggestion
  ${StyledInputPresentation} {
```